### PR TITLE
with-search depends on SearchSyntax

### PR DESCRIPTION
**Pull request by HeikoTheissen**

### CODE REVIEW

**Comment by ralfhandl** [comment]
`tools/V2-to-V4-CSDL.xsl` should now translate `sap:searchable="true"` into "Searchable=false, SearchSyntax=...Link to SAP search syntax..."

File: lib/csdl2openapi.js [hunk 1]
```diff
@@ -1,2905 +1,2908 @@
...
+    const searchRestrictions =
+      restrictions.SearchRestrictions ||
+      (target && target[voc.Capabilities.SearchRestrictions]) ||
+      {};
+
+    if (
+      searchRestrictions.Searchable !== false ||
+      searchRestrictions.SearchSyntax
+    ) {
+      if (searchRestrictions[voc.Core.Description]) {
```
Comment (ralfhandl) [inline lib/csdl2openapi.js:1007]:
```suggestion
      if (searchRestrictions[voc.Core.Description] || searchRestrictions.SearchSyntax) {
```
Produce "custom" parameter description also for custom search syntax and don't reference the reuse parameter which in turn links to the OData standard search syntax.

**Comment by ralfhandl** [comment]
@HeikoTheissen Do we need this PR?

**Comment by HeikoTheissen** [comment]
No longer needed since `Searchable: true` will remain.

**Pull request closed without merging**